kfdef/kfctl_existing_arrikto.yaml

```diff
@@ -309,4 +313,5 @@ spec:
     name: seldon-core-operator
   repos:
   - name: manifests
-    uri: https://github.com/kubeflow/manifests/archive/master.tar.gz
+    # uri: https://github.com/kubeflow/manifests/archive/master.tar.gz
+    uri: file:///home/yannis/go/src/github.com/kubeflow/manifests
```

Comment:
nit: Replace with master

Reply:
Haha I keep forgetting this. Thanks for the catch :smile: